src/react/features/filters/hooks/useTurnOnFilters.js

```diff
@@ -0,0 +1,23 @@
+import { useDispatch, useStore } from '../../../store'
+
+import { setFocus } from '../../navigation/actions'
+import { addDiory } from '../../room/actions'
+
+const SEARCH_TOOL_ID = 'SEARCH_TOOL_ID'
```

Comment:
This is defined in two places (also in useTextFilter). 

I don't know what is the correct place but this is incorrect! :D